demos/custom-tabs-headers-demo/src/main/java/org/chromium/customtabsdemos/MainActivity.java

```diff
@@ -0,0 +1,71 @@
+package org.chromium.customtabsdemos;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.browser.customtabs.CustomTabsCallback;
+import androidx.browser.customtabs.CustomTabsClient;
+import androidx.browser.customtabs.CustomTabsIntent;
+import androidx.browser.customtabs.CustomTabsService;
+import androidx.browser.customtabs.CustomTabsServiceConnection;
+import androidx.browser.customtabs.CustomTabsSession;
+import android.content.ComponentName;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.Browser;
+
+public class MainActivity extends AppCompatActivity {
+
+    private CustomTabsSession mSession;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        String url = "https://padr31.github.io/";
+
+        findViewById(R.id.btnExtra).setOnClickListener(e -> {
+
+            // Set up a callback that launches the cross origin intent after session validated.
+            CustomTabsCallback callback = new CustomTabsCallback() {
+                @Override
+                public void onRelationshipValidationResult(int relation, @NonNull Uri requestedOrigin, boolean result, @Nullable Bundle extras) {
```

Comment:
We limit lines to 100 chars long. Would you mind breaking it? There are other places where the same happen, can you check those too?